BenadrylAutomation/src/main/java/com/jnj/actions/Action.java

```diff
@@ -1004,30 +1004,16 @@ public static void navigateBack() {
 	public static void navigateForward() {
 		driver.navigate().forward();
 	}
-
+	
 	/**
 	 * Function to verify page url
-	 * 
 	 * @param partialUrl pass partial url to verify
 	 */
 	public static void verifyPageUrl(String partialUrl) {
 		String currentUrl = driver.getCurrentUrl().trim();
-		extentInfoLog("Asserting current URL : ", currentUrl);
+        extentInfoLog("Asserting current URL : ", currentUrl);
 		String expectedUrl = baseURI + partialUrl;
 		extentInfoLog("Expected URL : ", expectedUrl);
 		Assert.assertEquals(currentUrl, expectedUrl);
-		BaseClass.extentInfoLog("Verified current URL to be : " + currentUrl);
-	}
-
-	/**
-	 * Function to hover over element using actions class
-	 * 
-	 * @param driver  pass driver object
-	 * @param element pass webelement to hover over
-	 * @throws InterruptedException
-	 */
-	public static void hoverOverElement(WebDriver driver, WebElement element) throws InterruptedException {
-		Actions action = new Actions(driver);
-		action.moveToElement(element).perform();
 	}
```

Comment:
@zahinSJI why did you removed hoverOverElement() function?? Is it creating problem??

Reply:
Function is present in my code
![image](https://github.com/sji-autobots/BenadrylAutomation/assets/107922907/75cf4d3e-a730-431a-80e0-173abd75cac0)